bip-0174.mediawiki

```diff
@@ -322,6 +322,13 @@ The Transaction Extractor does not need to know how to interpret scripts in orde
 
 A single entity is likely to be both a Transaction Extractor and an Input Finalizer.
 
+==Encoding==
+
+A PSBT can be represented in two ways: in binary (as a file) or as a Base64 string using the encoding described in [https://tools.ietf.org/html/rfc4648#section-4 RFC4648].
+
+Binary PSBT files should use the <tt>.psbt</tt> file extension.
+The MIME type <tt>application/psbt</tt> can be used.
```

Comment:
I don't think you can just start using a mime type without registration.

Practices around mime types are defined in https://tools.ietf.org/html/rfc6838, but I think it may be too early to go through the registration processes. I'm not sure what to do here, but perhaps just write "A mime type name will be added to this document after blabla"?

Reply:
Done